src/targeting/build-page-targeting.ts

```diff
@@ -71,7 +71,8 @@ const filterValues = (pageTargets: Record<string, unknown>) => {
 const buildPageTargeting = (
 	consentState: ConsentState,
 	adFree: boolean,
-): PageTargeting => {
+	pfp: boolean,
```

Comment:
On a similar topic, now this function takes two boolean arguments, I was wondering if this would benefit from an object of named parameters?

Calling something like

```typescript
buildPageTargeting({ consentState, adFree: true, youtube: false });
```

instead of

```typescript
buildPageTargeting(consentState, true, false);
```

Reply:
Agreed on both fronts, have made the change!